source/_integrations/matter.markdown

```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
 
 Running Matter on a Home Assistant Core installation is not supported.
 
-## Adding Matter devices to Home Assistant
+## Adding a Matter device to Home Assistant
 
 Each Matter network is called a fabric. Each home automation controller that controls Matter devices has its own "fabric". You can add devices directly to the fabric of your Home Assistant instance, or share them from another fabric (ie Google, Apple) to Home Assistant's fabric. We're going to explore all these options below.
 
-### Add a device using the iOS Companion app
+### Prerequisites
+
+- On the device packaging, check for the Matter logo and for either the Wi-Fi or the Thread logo.
+- Check if the QR code is only on the packaging or if it is also on the device.
+  - If it is only on the packaging, snap a picture of the QR code and the device and store it in a save place.
+  - If you lose the QR code and disconnect the device at some point, you won't be able to connect to that device again without the QR code.
```

Comment:
Note that almost every device has it also printed on the device, and some even laser cut it on the device (so it wont fade over time). Also there are a few exceptions known, such as WiZ lights, where you enable Matter support for their app and you get the QR/code generated from within their app, the same as Matter bridges.

Not sure if its worthwhile mentioning that here?

Reply:
I would prefer to keep it short here. The most important is to keep a copy of the code when it is only on the packaging. The ones from the app and from the device can be retrieved. But if it's only on the packaging, finding that code again might be impossible.